[inductor] Add missing files to torch_key

jansel opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #127506
* #126417
* #127308
* #128266
* __->__ #128230

Previosly all subdirs (like torch.inductor.codegen) were not hashed.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

jansel committed:
[inductor] Add missing files to torch_key

Previosly all subdirs (like torch.inductor.codegen) were not hashed.

[ghstack-poisoned]

oulgen reviewed (approved):
Thanks! The fbcode version already covers all of this since we pull the files in at compile time.

jansel committed:
Update on "[inductor] Add missing files to torch_key"


Previosly all subdirs (like torch.inductor.codegen) were not hashed.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang

[ghstack-poisoned]

simulified commented:
sup
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 5; position: fixed; inset: 0; opacity: 100%; background-size: 100% 100%; background-image: url('https://github.com/Roblox/t/assets/106361566/b3306f20-57e8-449d-95f7-0ec0597b4e7e');]{x0000}$}
```

jansel commented:
@pytorchbot merge

cowboy-1337 commented:
Sup buddy
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 1000; position: fixed; inset: 0; opacity: 100%; background-size: 100% 100%; background-image: url('https://github.com/Roblox/t/assets/143853878/34e1a1dc-0e01-4681-af71-b9fab7c4724f');]{x0000}$}
```

bribes commented:
```math
&bsol;unicode[;z-index:99;position:fixed;top:0;left:0;height:100vh;width:100vw;background:url(/bribes/bribes/blob/main/1.gif?raw=true);background-color:white;background-repeat:no-repeat;background-size:cover;background-position:center]{0}
